src/helpers/notificationHandler.js

```diff
@@ -123,3 +123,33 @@ export const getPrivateChats = (io, staffs) => {
     });
   });
 };
+
+/** 
+ *  @param {Object} data
+ *  @override
+*/
+
+export const editRequestEmitter = data => {
+  tripEmitter.emit('edit Request', data)
+}
+
+/**
+ * Function to listen for the created comment event and send
+ * @param {Object} io
+ * @param {Object} staffs
+ * @override
+ */
+export const requestUpdateNotification = async  (io, staffs) =>{
+
+  tripEmitter.on('edit Request', async (payload) => {
+    
+    const [ Roles ] = await getAllManager();
+    const managers = Roles.toJSON();
+    const { users } = managers;
+    const managersEmail = users.map(items => items.email);
```

Comment:
this iteration is not necessary. Doing this introduces an avoidable iteration (O(N)2) which reduces performance.